apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/GraphQLCompiler.kt

```diff
@@ -189,13 +163,13 @@ class GraphQLCompiler(val logger: Logger = NoOpLogger) {
   private fun idClassName(generateKotlinModels: Boolean) = if (generateKotlinModels) {
     String::class.asClassName().toString()
   } else {
-    ClassNames.STRING.toString()
+    TODO("ClassNames.STRING.toString()")
```

Comment:
Is this change here by mistake?

Reply:
This is the java codegen path. It's currently broken in `dev-3.x` and will have to be enabled again.